README.md

```diff
@@ -51,3 +51,16 @@ any of the arguments passed to those loggers is a function, it will be called to
 get the value to be logged, however this will only happen if given logger is
 enabled. This way we can avoid paying for pre-processing if the value would not
 have been logged anyway.
+
+
+# Publishing to our npm registry
+
+Things to keep check before publishing:
+- any newly added source files which are to be included in the package have been added to "files" list in `package.json`
+- `version` in `package.json` has been updated appropriately
+
+From nix shell:
+
+```sh
+npm-publish
```

Comment:
can we also add a comment that a token must be populated in `netrc` file with publish permissions to enable this action?

Reply:
done in [dc96ce2](https://github.com/rhinofi/logger/pull/20/commits/dc96ce20db05507dd36006178561748510a59af2)